filer/private/sass/settings/_custom.scss

```diff
@@ -47,6 +48,8 @@ $screen-mobile: 320px;
 $screen-tablet: 720px;
 $screen-desktop: 975px;
 
+$screen-filer: 810px;
```

Comment:
I had to add a separate value for filer because height will break if I choose screen tablet.